### ISSUE TITLE
Fix go get errors 

### ISSUE THREAD

**Issue by iennae** [getporter/porter#1023]
## Describe the bug

Running go get currently leads to an error due to a missing " 

```
$ go get -u get.porter.sh/porter/cmd/porter
package get.porter.sh/porter/pkg/storage/pluginstore: /Users/sigje/go/src/get.porter.sh/porter/pkg/storage/pluginstore/doc.go:1: cannot parse import comment
package get.porter.sh/porter/pkg/secrets/pluginstore: /Users/sigje/go/src/get.porter.sh/porter/pkg/secrets/pluginstore/doc.go:1: cannot parse import comment
```
## To Reproduce
1. Run `go get -u get.porter.sh/porter/cmd/porter`

## Expected behavior
No errors when using go get.

**Comment by iennae** [comment]
Based on https://github.com/golang/go/issues/26367#issuecomment-405666468 I'm wondering if part of the solution here is to remove the import comments. I was trying to find whether there was a tool that could lint for these kind of issues and discovered that go issue.

**Comment by carolynvs** [comment]
Related issue that should fix this is #1285

**Comment by carolynvs** [comment]
Closing since #1603 was merged
